Assets/_Scripts/PlayerController.cs

```diff
@@ -46,6 +75,18 @@ void Flip()
         _facingRight = !_facingRight;
     }
 
+    private bool touchingCreature(){
+        Collider2D[] allCollisions = new Collider2D[10];
+        int count = playerCollider.OverlapCollider(contactFilter, allCollisions);
+
+        for (int i = 0; i< count; i++){
+            if (allCollisions[i].CompareTag("creature")){
```

Comment:
![image](https://github.com/user-attachments/assets/0ec13a25-952a-4356-9a8b-6b4c9c0d4732)

(assuming that this isnt currently a work in progress)